### PR TITLE
use try-with-resource

### PR DESCRIPTION
**Pull request by jukzi**

### CODE REVIEW

File: tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java
Comment (BeckerWdf) [inline tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java:1]: Changes in this file are super hard to to read on github.

**Review by BeckerWdf — APPROVED** [review]

File: tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java
Comment (jukzi) [reply tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java:1]: yes. i should be only a tab indent, maybe some white spaces where changed.

**Review by BeckerWdf — APPROVED** [review]

File: tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java
Comment (BeckerWdf) [reply tools/bundles/org.eclipse.e4.tools.emf.ui/src/org/eclipse/e4/tools/emf/ui/internal/common/resourcelocator/TargetPlatformContributionCollector.java:1]: Ah. With "ignore whitespace" it's readable.

**Comment by jukzi** [comment]
ignoring random failing https://github.com/eclipse-platform/eclipse.platform.ui/issues/757

**Pull request merged**